src/modules/fetcher/__tests__/FetcherModule.spec.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import FetcherModule from '../FetcherModule'
+import NativeFetcherModule from '../NativeFetcherModule'
+import type { TargetFilePathsType } from '../FetcherModule'
+
+jest.mock('../NativeFetcherModule')
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('FetcherModule', () => {
```

Comment:
A test of the progress callback is missing here.
To make sure the coverage is oke you can use the Coverage functionality in IntelliJ. You can find it next to the Run and Debug button. ("Run with coverage")

Reply:
You can create a jest.fn() mock and assert that it has been called for example a hundred times or that the final call had the return value of 1.